### ISSUE TITLE
Dark mode color tweaks

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#10418]
Bits and pieces spotted while further reviewing the application of dark mode across the UI.

Bug fixes:

- [ ] Hex code for alert
- [ ] Focus hex code (follow-up Nick’s work)
- [ ] Rich text comment highlight
- [ ] Selected / hovered rows in listings use grey-800
- [ ] Remove white borders in switch
    - change circle border to surface-page
    - Background use darker grey (100 or 150)
- [ ] Info panel dividers grey 400
- [ ] Longer menus shadow
- [ ] Re-add dividers in page explorer menu – same borders as header (grey-500)
- [ ] “No” buttons – danger/critical 100
- [ ] Hover for icon buttons: white (icon-primary)
- [ ] Block picker use surface-tooltip (and/or lighter grey)
    - Check surface-menus usage elsewhere
- [ ] Footer actions remove border
- [ ] Circled plus add buttons switch to teal 100 on hover (InlinePanel)

Improvements:

- [ ] Switch login screen to same Wagtail logo as sidebar
    - Try existing bird but with circle behind?
- [ ] Userbar accessibility checker dialog use dark theme as-is
- [ ] Userbar toggle switch to sidebar wagtail bird for both themes

**Comment by zerolab** [comment]
@thibaudcolas could you include #10379 to this as well?

**Comment by benenright** [comment]
- [ ] Close icon in modal (I did wonder tho if we can update this to be the floating 'X' as in Figma)

![Image](https://github.com/wagtail/wagtail/assets/4907819/8d04c3aa-0f6b-44bf-8457-0342eb6fb494)

**Comment by laymonage** [comment]
- Time picker in `DateTimeField`: 
<img width="714" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/192488c3-f016-4b64-856f-15bcf9da0134">

- Time picker in `TimeField`: 
<img width="380" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/d9a99f52-4ca1-4f4e-939e-dd0465a34045">

**Comment by thibaudcolas** [comment]
Just a note that #10467 fixed most of the above. There are further improvements we’d want to make, so please keep specific issues coming in as comments here.

**Comment by laymonage** [comment]
There's an issue with Tagit's autocomplete:

![tagit-darkmode-issue](https://github.com/wagtail/wagtail/assets/6379424/d499670a-a28c-48e2-ade7-1a22e6a798d3)

**Comment by laymonage** [comment]
`FieldPanel(read_only=True)`:

<img width="924" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/0f8ee62e-54b0-43bb-ba4b-447998f6c4df">

**Comment by thibaudcolas** [comment]
##  “No” buttons – danger/critical 100

This will need further consideration as we change the button’s background color to `critical-50` on hover, which would be too low contrast with a text in `critical-100`. Options:

- Retain `critical-200` as text and border color on hover
- Change hover styles to use something else than bg color swap

Note for this to be done we’ll need to introduce theme-ware color tokens, so ideally we’d not add too many.

##  Dark theme applied to the userbar

This involves adding the `w-theme-{% admin theme %}` class in the userbar template where appropriate (toggle, accessibility checker dialog?), and:

- Fixing components which aren’t correctly implemented in dark mode (modal close button)

**Comment by laymonage** [comment]
@thibaudcolas @albinazs we should try to pick https://github.com/wagtail/wagtail/issues/10418#issuecomment-1613184670 before 5.1 is released (can be during feature freeze) – it's a new feature in 5.1 and we should not release it in a broken state

**Comment by thibaudcolas** [comment]
@laymonage from what I can see the tagit issue has already been picked up. Screenshot of latest `main`:

![tagit-10418](https://github.com/wagtail/wagtail/assets/877585/cd0c9e94-291b-4914-b703-c550464a19e6)

I’ve opened #10689 to tackle the read-only FieldPanel issue.

**Comment by laymonage** [comment]
@thibaudcolas yep, tagit was fixed in #10601. I'll take a look your fix for the read-only FieldPanel. Thanks!

**Comment by thibaudcolas** [comment]
Looks like this is all done, closing.
